src/sentry/processing/realtime_metrics/project_budget_pb2.pyi

```diff
@@ -0,0 +1,29 @@
+from google.protobuf import descriptor as _descriptor
```

Comment:
same for this file -- there are tools to generate the `.pyi` stubs for you from protobufs and we should be using them in a client library rather than vendoring the client here